packages/nodes-base/nodes/Zoom/MeetingDescription.ts

```diff
@@ -0,0 +1,750 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const meetingOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'meeting',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a meeting',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a meeting',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve a meeting',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Retrieve all meetings',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a meeting',
+			}
+		],
+		default: 'create',
+		description: 'The operation to perform.',
+	}
+] as INodeProperties[];
+
+export const meetingFields = [
+	/* -------------------------------------------------------------------------- */
+	/*                                 meeting:create                                */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'User Id',
+		name: 'userId',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				operation: [
+					'create',
+				],
+				resource: [
+					'meeting',
+				],
+			},
+		},
+		description: 'User ID or email address of user.',
+	},
+	{
+		displayName: 'Additional settings',
+		name: 'additionalFields',
+		type: 'collection',
+		placeholder: 'Add Field',
+		default: {},
+		displayOptions: {
+			show: {
+				operation: [
+					'create',
+
+				],
+				resource: [
+					'meeting',
+				],
+			}
+		},
+		options: [
+			{
+				displayName: 'Agenda',
+				name: 'agenda',
+				type: 'string',
+				default: '',
+				description: 'Meeting agenda.',
+			},
+			{
+				displayName: 'Alternative Hosts',
+				name: 'alternative_hosts',
+				type: 'string',
+				default: '',
+				description: 'Alternative hosts email ids.',
+			},
+			{
+				displayName: 'Auto recording',
```

Comment:
Some fields use capitals in all initials; others use capitals only in the first initial. We should find out which we should be using.